cpp/src/gandiva/precompiled/arithmetic_ops_test.cc

```diff
@@ -101,4 +101,10 @@ TEST(TestArithmeticOps, TestDiv) {
   context.Reset();
 }
 
+TEST(TestArithmeticOps, TestRound) {
+  EXPECT_EQ(round_int32(758, -1), 760);
```

Comment:
test cases should cover negative numbers, and boundary conditions (i.e. .5 5).

Reply:
Added more test cases for negative numbers.